### ISSUE TITLE
Merge into upstream?

### ISSUE THREAD

**Issue by jkroepke** [thomasdarimont/keycloak-health-checks#18]
Hi @thomasdarimont ,

this Keycloak extensions looks great in combination with an kubernetes probe.

I saw you are already a Keycloak contributer. What did you think about to merge this "extension" into Keycloak upstream?

I guess a lot of people would benefit from + you are not responsible as one man show anymore.

**Comment by thomasdarimont** [comment]
@jkroepke Yes I already asked for this a few years ago, but got this topic got stuck.

I just started another thread on the keycloak-dev mailing list, feel free to join the discussion :)
https://groups.google.com/g/keycloak-dev/c/vVr2PJ8qYBQ
